client/modules/datafiles/src/DatafilesToolbar/DatafilesToolbar.tsx

```diff
@@ -86,13 +86,25 @@ export const DatafilesToolbar: React.FC<{ searchInput?: React.ReactNode }> = ({
       return {
         canPreview:
           selectedFiles.length === 1 && selectedFiles[0].type === 'file',
-        canRename: user && selectedFiles.length === 1 && !isReadOnly,
-        canCopy: user && selectedFiles.length >= 1,
-        canTrash: user && selectedFiles.length >= 1 && !isReadOnly,
+        canRename:
+          user &&
+          selectedFiles.length === 1 &&
+          !isReadOnly &&
+          !selectedFiles[0].path.endsWith('.hazmapper'),
```

Comment:
Any reason you don't want to be able to do these actions with hazmapper files?

Reply:
This is the current functionality in prod and it breaks the linking between a project and a hazmapper map. It's a hazmapper functionality requirement - I'm trying to find the docs on why specifically this is the case